### PR TITLE
12.0 merge

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_orders_history_reprint/doc/changelog.rst [hunk 1]
```diff
@@ -1,7 +1,14 @@
-.. _changelog:
+`1.0.6`
+-------
+
+- **Fix:** Downloading tickets even if the "Reprint" option is off
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history_reprint/doc/changelog.rst:4]: :+1:

File: pos_orders_history_reprint/doc/changelog.rst [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+- **Fix:** "Reprint" button on the orders history list was shown when "Reprint" option was off
```
Comment (Ramil-Mukhametzyanov) [inline pos_orders_history_reprint/doc/changelog.rst:5]: :+1:

File: pos_orders_history_reprint/doc/changelog.rst [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
+
+`1.0.5`
+--------
+
+- **Fix:** Correcting "There is No Ticket for the Order" Message for Orders with Invoiced Status
```
[the remark this answers is not in the document]
Comment (Ramil-Mukhametzyanov) [reply pos_orders_history_reprint/doc/changelog.rst:10]: Unable to pay invoice in POS. And can not close the POS by clicking the button in the upper right corner.

**Review by Ramil-Mukhametzyanov — CHANGES_REQUESTED** [review]

**Review by Ramil-Mukhametzyanov — APPROVED** [review]
It was a problem with the network.

**Comment by KolushovAlexandr** [comment]
@Ramil-Mukhametzyanov I found out the reason for the problem
https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/models/account_invoice.py#L19
this line uses * _compute_average_price * it's the method added in the module `sale_stock`, but the latter module is not presented in the dependencies for `point_of_sale`. The error does not appear if the `sale_stock` module is installed

File: pos_qr_scan/doc/changelog.rst [hunk 4]
```diff
@@ -1,3 +1,8 @@
+`1.0.2`
+-------
+
+- **Fix:** - Scanner didn't work due to "ReferenceError: gCtx is not defined"
+
```
[the remark this answers is not in the document]
Comment (em230418) [reply pos_qr_scan/doc/changelog.rst:5]: Работает.

**Review by em230418 — APPROVED** [review]

**Comment by em230418** [comment]
I approve to merge it now

**Pull request merged**